Readme.md

```diff
@@ -56,6 +56,10 @@ make develop
 
 When switching key pairs on a channel, the new certficiate must be built into the bundle, which must then be signed with the old key. For this purpose, the `--override-cert` option of the deploy script is needed to provide RAUC with a certificate matching the new key.
 
+## Change Log
+
+Update the change log for every release. See http://keepachangelog.com/ for formatting and conventions.
```

Comment:
Would this be a good idea be able to view PlayOS’s change log from the controller ?

Reply:
Nice to have, yes.

Reply:
I added a card.